lib/correios/cep/address.ex

```diff
@@ -23,6 +23,56 @@ defmodule Correios.CEP.Address do
   The values will be converted to string, and `complement` and `complement2` keys will be joined
   into the `complement` field.
 
+  ## Examples
+
+      iex> #{inspect(__MODULE__)}.new(%{
+      ...>   street: 'Street',
+      ...>   neighborhood: 'Neighborhood',
+      ...>   complement: ' complement one',
+      ...>   complement2: 'complement two ',
+      ...>   city: 'City',
+      ...>   state: 'ST',
+      ...>   postal_code: '12345678'
+      ...> })
+      %#{inspect(__MODULE__)}{
+        city: "City",
+        complement: "complement one complement two",
+        neighborhood: "Neighborhood",
+        state: "ST",
+        street: "Street",
+        postal_code: "12345678"
+      }
+
+  """
+  @spec new(map()) :: t()
+  def new(%{
+        street: street,
+        complement: complement1,
+        complement2: complement2,
+        neighborhood: neighborhood,
+        city: city,
+        state: state,
+        postal_code: postal_code
+      }) do
+    %__MODULE__{
+      street: to_string(street),
+      neighborhood: to_string(neighborhood),
+      city: to_string(city),
+      state: to_string(state),
+      postal_code: to_string(postal_code),
+      complement: build_complement(complement1, complement2)
+    }
+  end
+
+  @doc """
+  Creates a new `#{inspect(__MODULE__)}` struct with the given values.
```

Comment:
No need to duplicate the `new/1`, since `zipcode` key will never be present in the input params. 

https://github.com/prodis/correios-cep-elixir/blob/f548949ff57bf774777cd328aabc9ccb612681be/lib/correios/cep/parser.ex#L17

Reply:
I'll fix it.